src/lib/modernExtend.ts

```diff
@@ -465,6 +468,8 @@ export function numeric(args: NumericArgs): ModernExtend {
                 if (scale !== undefined) {
                     value = typeof scale === 'number' ? value / scale : scale(value, 'from');
                 }
+                assertNumber(value);
+                if (precision) value = precisionRound(value, precision);
```

Comment:
```suggestion
                if (precision != null) value = precisionRound(value, precision);
```

Please change to this, otherwise `precision` `0` won't work.

Reply:
But it works, I tested this function in a sandbox with a floating point number and different `precision`. Precision `0` returns integer as expected.